tensorflow/compiler/tf2tensorrt/convert/convert_nodes.h

```diff
@@ -508,30 +508,20 @@ Status GetTrtBroadcastShape(const TRT_TensorOrWeights& operand_l,
                             nvinfer1::Dims* operand_r_new_dims);
 
 // Map of all supported UnaryOperations
-const std::unordered_map<string, nvinfer1::UnaryOperation>* UnaryOperationMap();
+template <typename T>
+using operationMap = std::unordered_map<std::string, T>;
+
+typedef operationMap<nvinfer1::UnaryOperation> unaryOperationMap;
```

Comment:
Let's move the comment in line 510 to here.
Would you please also add a period to the comments in line 510, 516, 518? I understand those are old code.